src/lib/components/card-v2/card-v2.svelte

```diff
@@ -218,6 +218,14 @@
   </ul>
 </div>
 
+<div
+  data-popup="card-synchronization-{snippet.id}"
+  class="z-10 card w-80 p-4 variant-filled-warning"
+>
+  Local data and remote data conflicted.
+  Please click to compare and resolve!
+</div>
```

Comment:
One thing to do now: toggle the visibility of the button by synchronization state, and the corresponding merge modal.